app/controllers/chefs/ingredients_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Chefs::IngredientsController < ApplicationController
+
+  def index
+    @chef = Chef.find(params[:id])
+    @ingredients = @chef.ingredients.all
+  end
+end
```

Comment:
I'm curious, Are `@chef.ingredients.all` and `@chef.ingredients` the same thing?

I'm also wondering why you set `@ingredients` to `@chef.ingredients` since you ended up doing `@chef.ingredients` in the view anyway. I think it would have been easier in the view to just use the `@ingredients` variable you created in the controller.